### DISCUSSION TITLE
Welcome to DCE-DSC-MRI_CodeCollection Discussions!

### DISCUSSION THREAD

**Discussion by stadmill** [OSIPI/DCE-DSC-MRI_CodeCollection#113] [General]
## 👋 Dear TF2.3 Members,

  Welcome!
  We’re using Discussions as a place to connect with other members of our task force. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

 @LucyKershaw and I have taken over leadership for the next two years from @petravanhoudt and @mjt320, who have done an excellent job kicking off TF2.3. We are very much looking forward to working with you all.

**Comment by petravanhoudt** [comment]
Nice, hope this works out useful! Might be good to put a message in the slack channel to notify people that discussions are here now.

**Reply by stadmill** [reply]
Good point. Done.

**Comment by fedorov** [comment]
Excellent idea to use GitHub discussions in place of Slack! 👍 💯

**Comment by plaresmedima** [comment]
Agree - whoever does not get those is not in the right place anyway!

Prof. Steven Sourbron, PhD
Chair in Medical Imaging Physics
University of Sheffield, UK
*https
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>*
*Sheffield abdominal imaging research page*
<https://www.sheffield.ac.uk/medicine/research/research-themes/medical-imaging/medical-imaging-research/abdominal-imaging>
*Open source initiative in perfusion imaging* <http://www.osipi.org>
*Magnetic resonance imaging biomarkers for chronic kidney disease
<https://renalmri.org/>*

**Comment by stadmill** [comment]
The survey for members is still open here https://protect-au.mimecast.com/s/hJnHCnx1jniGk54xEF9nSC4?domain=forms.gle

**Comment by sudarshanragu** [comment]
Congratulations @stadmill and @LucyKershaw ! Looking forward to continue to contribute to the task force and love the idea of using Discussions ! :)
